client/cli/src/params/shared_params.rs

```diff
@@ -19,8 +19,10 @@
 use sc_service::config::BasePath;
 use std::path::PathBuf;
 use structopt::StructOpt;
+use crate::logger::LogRotationOpt;
 
 /// Shared parameters used by all `CoreParams`.
+#[allow(missing_docs)]
```

Comment:
Don't put that on the whole struct otherwise it will not warn if people add fields and forget the doc! This needs to be placed right before the field you want to ignore doc so it doesn't interfere with the other fields

Reply:
Fixed by https://github.com/paritytech/substrate/pull/6608/commits/52710e0191046f2b17584b69b2b1baa151488df3